lib/my_inject.rb

```diff
@@ -0,0 +1,37 @@
+class MyArray < Array
+
+  def my_inject (initial = self.shift, sym=nil, &block)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis.